pkg/api/handlers/compat/system.go

```diff
@@ -90,11 +90,10 @@ func GetDiskUsage(w http.ResponseWriter, r *http.Request) {
 	}
 
 	utils.WriteResponse(w, http.StatusOK, handlers.DiskUsage{DiskUsage: docker.DiskUsage{
-		LayersSize:  df.ImagesSize,
-		Images:      imgs,
-		Containers:  ctnrs,
-		Volumes:     vols,
-		BuildCache:  []*docker.BuildCache{},
-		BuilderSize: 0,
+		LayersSize: df.ImagesSize,
+		Images:     imgs,
+		Containers: ctnrs,
+		Volumes:    vols,
+		BuildCache: []*docker.BuildCache{},
```

Comment:
I mean practically speaking this is still the same as go will initialize them with the zero value with is 0.
So maybe a comment here that we do not have a BuilderSize equivalent would better for future readers of the API code?

Reply:
ok I didn't know that go initializes the value as 0. but I'll put a comment and push it. also I'll fix the test that, I thought, didn't fail locally, but probably need to double-check